src/_backend_agg.cpp

```diff
@@ -45,7 +45,7 @@ RendererAgg::RendererAgg(unsigned int width, unsigned int height, double dpi)
       rendererBase(),
       rendererAA(),
       rendererBin(),
-      theRasterizer(8192),
+      theRasterizer(32768),
```

Comment:
What's the meaning of this change?

Reply:
And does it need documenting in an API change note?

Reply:
That is:
> - makes the upper limit higher so it will be hit less often